### DISCUSSION TITLE
Support WinGet.

### DISCUSSION THREAD

**Discussion by neko3cs** [raphamorim/rio#519] [Ideas]
WinGet is a default Windows package manager.
So I think rio should support WinGet rather than Chocolatey.
Of course, it is awesome that support Chocolatey.

**Comment by neko3cs** [comment]
This is how to upload WinGet package.

🔗[Create your package manifest | Microsoft Learn](https://learn.microsoft.com/en-us/windows/package-manager/package/manifest?WT.mc_id=DOP-MVP-4039781%3Ftabs%3Dminschema%2Cversion-example&tabs=minschema%2Cversion-example)

**Comment by raphamorim** [comment]
Hey @neko3cs I think is a good idea! Will be moving it for issues so I can keep a track on it better.

**Reply by neko3cs** [reply]
Thank you for your empathy!
I will look forward to do it.

**Comment by raphamorim** [comment]
supported on #520

**Reply by neko3cs** [reply]
Thank you soooo much! :)
I installed it!
